onnxruntime/core/session/provider_bridge_ort.cc

```diff
@@ -1074,8 +1074,8 @@ struct ProviderHostImpl : ProviderHost {
 #endif
 
 #if defined(USE_CUDA) || defined(USE_ROCM)
-
   PhiloxGenerator& PhiloxGenerator__Default() override { return PhiloxGenerator::Default(); }
+#endif
```

Comment:
Can you explain why this change is needed?

Reply:
Otherwise, DORT won't build for CPU. I think this is an incorrect change introduced by `git merge`, `PhiloxGenerator__Default`, a kernel specific class, shouldn't be related python bridge code such as tracking python object's reference count.